src/Kovan-DssSpell.t.sol

```diff
@@ -853,7 +878,76 @@ contract DssSpellTest is DSTest, DSMath {
         assertEq(flipAAVEA.kicks(), 1);
     }
 
-    // TODO test Uni lp integration
+    // Test any Integrations
+
+    // TODO test Aave integration
+
+    function testSpellIsCast_UNIV2DAIETH_INTEGRATION() public {
+       vote();
+       scheduleWaitAndCast();
+       assertTrue(spell.done());
+
+       bytes32 ilk = "UNIV2DAIETH-A";
+
+       lpPip.poke();
+       hevm.warp(now + 3601);
+       lpPip.poke();
+       spot.poke(ilk);
+
+       // Check faucet amount
+       uint256 faucetAmount = faucet.amt(address(lp));
+       assertTrue(faucetAmount > 0);
+       faucet.gulp(address(lp));
+       assertEq(lp.balanceOf(address(this)), faucetAmount);
+
+       // Check median matches pip.src()
+       assertEq(lpPip.src(), address(lp)); // TODO: Check orbs
```

Comment:
This is failing. Is the oracle configured correctly?

```
  «Error: Wrong `address' value»
    Expected: 0xfd5608515a47c37afba68960c1916b79af9491d0
      Actual: 0xb10cf58e08b94480fcb81d341a63295ebb2062c2
```

Reply:
Oh....

The Kovan Oracle is using the actual UniV2 LP token on-chain, but we have a dummy token in it's place. This complicates things.

Reply:
I think we should use the real tokens in this case. We can generate DAI and we have plenty kovan ETH, so maybe this shouldn't be a dummy token at all, nor be in the faucet.

Reply:
Oops, I should have read your next comment before writing.